Client.xcodeproj/project.pbxproj

```diff
@@ -19812,10 +19824,13 @@
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
-				CODE_SIGN_STYLE = Automatic;
+				CODE_SIGN_ENTITLEMENTS = "";
+				CODE_SIGN_IDENTITY = "";
+				CODE_SIGN_STYLE = Manual;
 				COPY_PHASE_STRIP = NO;
 				CURRENT_PROJECT_VERSION = 1;
 				DEBUG_INFORMATION_FORMAT = dwarf;
+				DEVELOPMENT_TEAM = 33YMRSYD2L;
```

Comment:
This was moved to be handled by the config files on https://github.com/ecosia/ios-browser/pull/738... any reason to add it to the project here now?

Reply:
💯 ! The config file works for all the client-facing targets. For the `EcosiaSnapshotTests` I thought about not assigning any specific config so that that config wouldn't interfere with tests. This decision led to assign the dev team for this target.